packit_service/worker/parser.py

```diff
@@ -366,47 +366,34 @@ def get_gitlab_push_common_data(event) -> tuple:
         parsable in the same way.
 
         Returns:
-            a tuple like (actor, project_url, parsed_url, ref, head_commit, head_commit_sha)
+            a tuple like (actor, project_url, parsed_url, ref, head_commit)
         Raises:
             PackitParserException
         """
-        raw_ref = event.get("ref")
+        if not (raw_ref := event.get("ref")):
+            raise PackitParserException("No ref info from event.")
```

Comment:
Is this change intentional? Before it was just a warning.

Reply:
Yup, later we `split()` it so if this returned `None` we'd end up with an `AttributeError` later anyway, so rather stop parsing right now.

Thanks for the reviews.